pyproject.toml

```diff
@@ -31,6 +31,7 @@ sphinxcontrib-fulltoc = {version = "^1.2.0", extras = ["docs"]}
 Sphinx = {version = "^5.1.1", extras = ["docs"]}
 myst-parser = {version = "^0.18.0", extras = ["docs"]}
 sphinx-rtd-theme = {version = "^1.0.0", extras = ["docs"]}
+async-timeout = ">=4.0.1"
```

Comment:
We had a request in Bleak to also allow v3.x.

https://github.com/hbldh/bleak/pull/1009

so requiring >=4 would cause a regression in Bleak

Reply:
We don't need the 4.x api specifically so we can reduce the pin to 3.x

Reply:
https://github.com/Bluetooth-Devices/dbus-fast/pull/39

We pin it higher in hass to solve dep chain compat with aiohttp but that isn't a problem for `.timeout()` in `async` context so  no worries about loosening the pin here.